### PR TITLE
Updating target to v4.5

### PR DESCRIPTION
**Pull request by HenrikRxn**
Fixing #48

### CODE REVIEW

**Comment by HenrikRxn** [comment]
@parthopdas Please review

File: Hosts/VS/CommonUI/CommonUI.csproj [hunk 1]
```diff
@@ -47,7 +47,10 @@
     <CodeAnalysisIgnoreGeneratedCode>false</CodeAnalysisIgnoreGeneratedCode>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="FSharp.Core, Version=$(TargetFSharpCoreVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
+    <Reference Include="FSharp.Core, Version=$(TargetFSharpCoreVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\FSharp.Core.3.1.2.5\lib\net40\FSharp.Core.dll</HintPath>
```
Comment (parthopdas) [inline Hosts/VS/CommonUI/CommonUI.csproj]: Can we remove this hintpath please and specify Specificversion as true?

File: Hosts/VS/TddStud10App/packages.config [hunk 2]
```diff
@@ -3,6 +3,7 @@
   <package id="BuildTools.FxCop" version="1.0.1" targetFramework="net45" />
   <package id="CommonServiceLocator" version="1.3" targetFramework="net45" />
   <package id="EditorUtils2010" version="1.5.0.1" targetFramework="net45" />
+  <package id="FSharp.Core" version="3.1.2.5" targetFramework="net45" />
```
Comment (parthopdas) [inline Hosts/VS/TddStud10App/packages.config]: version is 4.3.1.0 right?

**Comment by parthopdas** [comment]
Lets move away from FSCore 3.1.2.5 and unify everything to 4.3.1.0?

File: Hosts/VS/TddStud10App/packages.config (hunk 2 above)
Comment (HenrikRxn) [inline Hosts/VS/TddStud10App/packages.config]: It is a little confusing, but the NuGet package refers to the F# version

I.e. v3.1.2.5 of the NuGet package is for F# 3.1, i.e. version 4.3.1.0 of FSharp.Core 

Version 4.0.0.1 of the NuGet package is for F# 4.0, i.e. version 4.4.0.0 of FSharp.Core

**Comment by HenrikRxn** [comment]
Repeated from line comment

It is a little confusing, but the NuGet package refers to the F# version

I.e. v3.1.2.5 of the NuGet package is for F# 3.1, i.e. version 4.3.1.0 of FSharp.Core

Version 4.0.0.1 of the NuGet package is for F# 4.0, i.e. version 4.4.0.0 of FSharp.Core

File: Hosts/VS/CommonUI/CommonUI.csproj (hunk 1 above)
Comment (HenrikRxn) [inline Hosts/VS/CommonUI/CommonUI.csproj]: Why?

**Comment by parthopdas** [comment]
Wow! I never knew that... I always wondered about the visible discrepancy, but never bothered to resolve it. My inline comment was related to the same confusion, so ignore it.

**Pull request merged**